.github/workflows/build.yml

```diff
@@ -224,7 +228,6 @@ jobs:
     if: ${{ github.event_name == 'push' && github.repository == 'winglang/wing' }}
     needs:
       - quality-gate
-      - build
```

Comment:
This is required because some of the steps in this job use `needs.build`. afaik `need`ing quality-gate isn't enough. What was the intention in removing it?

Reply:
Just the redundancy as quality gate already needs build. I was a bit skeptical about this actually. Since it is a necessity I'll add it back.